lib/convection/model/template/resource/aws_elbv2_listener.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::Listener
+        ##
+        class ELBV2Listener < Resource
+          type 'AWS::ElasticLoadBalancingV2::Listener', :elbv2_listener
+          property :certificates, 'Certificates', :type => :list
+          property :default_actions, 'DefaultActions', :type => :list
+          property :load_balancer_arn, 'LoadBalancerArn'
+          property :port, 'Port'
+          property :protocol, 'Protocol'
+          property :ssl_policy, 'SslPolicy'
+
+          def listener_certificate(&block)
+            certificate = ResourceProperty::ELBV2ListenerCertificates.new(self)
+            certificate.instance_exec(&block) if block
+            certificates << certificate
+          end
```

Comment:
I just re-checked the AWS docs:
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listener.html

and I think you're correct that it should just be named "certificate".  Sometimes, Amazon will use one thing as the json key, and the value is a different type (such as Targets vs TargetDescription at http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html) but that's not the case here.  Good catch.